build.gradle

```diff
@@ -106,10 +128,6 @@ sourceSets {
     }
 }
 
-tasks.withType(JavaCompile).all {
-    options.compilerArgs.add("-Xlint:all")
```

Comment:
It would be nice to turn on all lint warnings.

Reply:
Warnings are turned on. Maybe we should make it as error in next PR and also update CF version